apps/opik-backend/src/main/java/com/comet/opik/domain/SpanDAO.java

```diff
@@ -84,6 +92,10 @@ INSERT INTO spans(
                         :input<item.index>,
                         :output<item.index>,
                         :metadata<item.index>,
+                        :model<item.index>,
+                        :provider<item.index>,
+                        toDecimal32(:total_estimated_cost<item.index>, 7),
```

Comment:
To confirm, this interval allows -999.9999999, 999.9999999. Do product think this is enough?

Reply:
Also, we shouldn't round this here; We should handle it before persisting.

Reply:
Actually I missed and we need 8 digits precision, so it would be even less, up to 99.9999999.
It's enough, the most expensive price is $60 per 1M, and tokens limit is 128K tokens shared between prompt and completion. So potential max value would be ~$6.5. And as I checked prod db, it would be millicents in most cases.

For rounding I updated logic

Reply:
After all had to change to Decimal64 :) When increased precision to 8 digits Decimal32 started to fail.